### PR TITLE
SIP-116 Optimism standard deposit/withdrawal interface

### PR DESCRIPTION
**Pull request by ajsantander**
Proposal to update our bridges to conform to the standard interface being proposed by Optimism.

### CODE REVIEW

File: sips/sip-119.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+sip: 119
+title: Optimism standard deposit/withdrawal interface
+status: WIP
+author: Alejandro Santander (@ajsantander), Yannis Stamelakos (@i-stam)
+discussions-to: https://research.synthetix.io/t/optimism-standard-deposit-withdrawa-interface/322
+
+created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
```
Comment (justinjmoses) [inline sips/sip-119.md:8]: please add

File: sips/sip-119.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+requires (*optional): <SIP number(s)>
```
Comment (justinjmoses) [inline sips/sip-119.md:9]: please remove__

File: sips/sip-119.md (hunk 1 above)
Comment (ajsantander) [reply sips/sip-119.md]: Done

File: sips/sip-119.md (hunk 2 above)
Comment (ajsantander) [reply sips/sip-119.md]: Done

File: sips/sip-116.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+### Rationale
+<!--This is where you explain the reasoning behind how you propose to solve the problem. Why did you propose to implement the change in this way, what were the considerations and trade-offs. The rationale fleshes out what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+First, changes should be made to `SynthetixBridgeToOptimism` and `SynthetixBridgeToBase`. Following that, the corresponding test cases should be updated, and tests should be written to cover new functionality. Finally, the two contracts should be deployed on each layer, and linked to the system. The bridge in L1 is particularly sensitive because it locks a large number of funds.
+
+### Technical Specification
+<!--The technical specification should outline the public API of the changes proposed. That is, changes to any of the interfaces Synthetix currently exposes or the creations of new ones.-->
```
Comment (justinjmoses) [inline sips/sip-116.md:43]: Can you link to eth-optimism's interface?

Comment (ajsantander) [reply sips/sip-116.md:43]: Done

**Review by justinjmoses — APPROVED** [review]

**Pull request merged**